ddtrace/contrib/aiohttp/middlewares.py

```diff
@@ -0,0 +1,96 @@
+from ..asyncio import context_provider
+from ...ext import AppTypes, http
+from ...compat import stringify
+
+
+class TraceMiddleware(object):
+    """
+    aiohttp Middleware class that will append a middleware coroutine to trace
+    incoming traffic.
+    """
+    def __init__(self, app, tracer, service='aiohttp-web'):
```

Comment:
the ``aiohttp`` middleware must follow the framework guidelines so that users can:
```python
web.Application(
  middleware=[
    tracer_middleware,
    # other middlewares
  ]
)
```